src/sqllogictest/src/runner.rs

```diff
@@ -1043,6 +1041,8 @@ impl<'a> RunnerInner<'a> {
             deploy_generation: None,
             http_host_name: Some(host_name),
             internal_console_redirect_url: None,
+            // TODO(txn): Get this flipped to true before turning anything on in prod.
```

Comment:
This part is still key. Do we expect everything interesting to be covered by existing tests? I didn't follow this feature closely before.

Reply:
Yeah, I promise I'll remember this! The `TODO(txn)` marker is one I'm grepping for and making sure they're all resolved before we turn it on in prod.

We'll definitely want some new testing, but it's not quite ready for it yet. Hoping within the next week or so! I've promised Philip that I'll ping QA when it's ready, feel free to continue ignoring this until then